### PR TITLE
[FSTORE-309] Add support for like operator in query filter

### PR DESCRIPTION
**Pull request by bubriks**
This PR adds/fixes/changes...
- please summarize your changes to the code 
- and make sure to include all changes to user-facing APIs

JIRA Issue: -

Priority for Review: -

Related PRs: -

**How Has This Been Tested?**

- [ ] Unit Tests
- [ ] Integration Tests
- [ ] Manual Tests on VM

**Checklist For The Assigned Reviewer:**

```
- [ ] Checked if merge conflicts with master exist
- [ ] Checked if stylechecks for Java and Python pass
- [ ] Checked if all docstrings were added and/or updated appropriately
- [ ] Ran spellcheck on docstring
- [ ] Checked if guides & concepts need to be updated
- [ ] Checked if naming conventions for parameters and variables were followed
- [ ] Checked if private methods are properly declared and used
- [ ] Checked if hard-to-understand areas of code are commented
- [ ] Checked if tests are effective
- [ ] Built and deployed changes on dev VM and tested manually
- [x] (Checked if all type annotations were added and/or updated appropriately)
```

### CODE REVIEW

**Review by davitbzh — COMMENTED** [review]
@bubriks is it possible to include code snippets for this, either here or in docs?

File: java/src/main/java/com/logicalclocks/hsfs/Feature.java [hunk 1]
```diff
@@ -196,6 +196,10 @@ public Filter in(Collection<?> collection) {
     return new Filter(this, SqlFilterCondition.IN, jsonArray.toString());
   }
 
+  public Filter lk(Feature value) {
```
Comment (moritzmeister) [inline java/src/main/java/com/logicalclocks/hsfs/Feature.java:199]: Let's call it also `like` like in python?

**Review by moritzmeister — CHANGES_REQUESTED** [review]

File: python/hsfs/feature.py [hunk 2]
```diff
@@ -215,6 +215,9 @@ def __gt__(self, other):
     def contains(self, other):
         return filter.Filter(self, filter.Filter.IN, json.dumps(other))
 
+    def like(self, other):
```
Comment (moritzmeister) [inline python/hsfs/feature.py:218]: Should be possible to have a simple unit test for this?

File: java/src/main/java/com/logicalclocks/hsfs/Feature.java (hunk 1 above)
Comment (bubriks) [reply java/src/main/java/com/logicalclocks/hsfs/Feature.java:199]: Ok will change. I saw all methods were 2 char so I also made this like that.

Comment (moritzmeister) [reply java/src/main/java/com/logicalclocks/hsfs/Feature.java:199]: yeah but prefer conistency over short cause we don't have java docs

Comment (bubriks) [reply java/src/main/java/com/logicalclocks/hsfs/Feature.java:199]: should i also rename "in" for "contains" (in python it is "contains")?

**Review by moritzmeister — APPROVED** [review]

File: java/src/main/java/com/logicalclocks/hsfs/Feature.java (hunk 1 above)
Comment (moritzmeister) [reply java/src/main/java/com/logicalclocks/hsfs/Feature.java]: no I actually think "contains" is bad, `x contains ["avx"]` that sounds incorrect, or? the condition is actually `x in ["a", "b"]

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**